src/components/embedded-tabs/EmbeddedTabs.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+import { Box, BoxProps } from 'rebass';
+import * as S from './styles';
+import Tab, { Props as EmbeddedTabItem } from './Tab';
+import * as R from 'ramda';
+
+export interface Props extends Omit<BoxProps, 'css'> {
+  tabs: EmbeddedTabItem[];
+  initialTab?: number;
+  onTabChange?: (tabIndex: number) => void;
+}
+
+const EmbeddedTabs: FC<Props> = ({
+  tabs,
+  onTabChange,
+  initialTab = 0,
+  sx = {},
+  ...boxProps
+}: Props) => {
+  const [active, setActive] = useState(R.clamp(0, tabs.length - 1, initialTab));
```

Comment:
nit: maybe `activeIndex` would be a better name

Reply:
just curious, is there any specific reason to use clamp here? you don't want to let the developer make mistake when defining the initialTabeindex?

Reply:
changed to `activeindex`.

On a second thought, the clamp is not good. I think it should fail. Removed it.